project/src/components/displaying-movie-card/displaying-movie-card.tsx

```diff
@@ -0,0 +1,14 @@
+function DisplayingMovieCard(): JSX.Element {
```

Comment:
названия компонентов не надо начинать с глагола, только методы

```suggestion
function MovieCard(): JSX.Element {
```